lighthouse-core/audits/webapp-install-banner.js

```diff
@@ -73,25 +73,30 @@ class WebappInstallBanner extends MultiCheckAudit {
     }
   }
 
-  static assessOfflineStartUrl(artifacts, failures) {
-    const hasOfflineStartUrl = artifacts.StartUrl === 200;
+  static assessOfflineStartUrl(artifacts, failures, result) {
+    const hasOfflineStartUrl = artifacts.StartUrl.statusCode === 200;
+
     if (!hasOfflineStartUrl) {
       failures.push('Manifest start_url is not cached by a Service Worker');
+
+      if (artifacts.StartUrl.debugString) {
+        failures.push(artifacts.StartUrl.debugString);
+      }
+    } else if (artifacts.StartUrl.debugString) {
+      result.debugString = artifacts.StartUrl.debugString;
     }
   }
 
   static audit_(artifacts) {
     const failures = [];
 
     return artifacts.requestManifestValues(artifacts.Manifest).then(manifestValues => {
-      WebappInstallBanner.assessManifest(manifestValues, failures);
-      WebappInstallBanner.assessServiceWorker(artifacts, failures);
-      WebappInstallBanner.assessOfflineStartUrl(artifacts, failures);
-
-      return {
-        failures,
-        manifestValues
-      };
+      const result = {failures, manifestValues};
+      WebappInstallBanner.assessManifest(manifestValues, failures, result);
+      WebappInstallBanner.assessServiceWorker(artifacts, failures, result);
+      WebappInstallBanner.assessOfflineStartUrl(artifacts, failures, result);
```

Comment:
`MultiCheckAudit` is already somewhat confusing and this doesn't exactly simplify things :)

Short of a `MultiCheckAudit` refactor, what about something like a `warnings` array, which is used for `debugString`, just like `failures` but they won't fail the audit?

Just an idea, maybe we can come up with something simpler/more self-explanatory